.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,91 @@
+name: Bug Report
+description: Found something you weren't expecting?  Report it here!
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
+- type: markdown
+  attributes:
+    value: |
+      1. Please speak English, this is the language all maintainers can speak and write.
+      2. Please ask questions or configuration/deploy problems on our Discord
+         server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
+      3. Please take a moment to check that your issue doesn't already exist.
+      4. Make sure it's not mentioned in the FAQ (https://docs.gitea.io/en-us/faq)
+      5. Please give all relevant information below for bug reports, because
+         incomplete details will be handled as an invalid report.
+- type: input
+  id: gitea-ver
+  attributes:
+    label: Gitea Version
+    description: Gitea version (or commit reference) your instance is running
+    placeholder: v1.14.3
+    value: v1.14.3
```

Comment:
I don't think we should have a default value. And maybe not a placeholder either. I fear users will leave the default value unchanged, without confirming they are actually using that version.

Reply:
Okay, I can live with both. I would have liked it, but I can also see your concern…

Reply:
I removed the placeholder and default version in my recent commit.